corehq/apps/settings/views.py

```diff
@@ -205,9 +215,11 @@ def form_actions(self):
 
     @property
     @memoized
+    @two_factor_exempt
```

Comment:
You should only need the decorator on the `dispatch` method I think.

Reply:
Fixed here:
https://github.com/dimagi/commcare-hq/pull/19570/commits/6fa949855daef5a6098e132b6e4d2382d812944a